lib/editor/components/map/PatternLocationMarker.js

```diff
@@ -110,44 +113,48 @@ export default class PatternLocationMarker extends Component<Props> {
             fillColor={active ? 'blue' : '#666'}
             fillOpacity={0.25}
             icon={patternStopIcon}
-            key={patternLocation.id}
+            key={patternHaltIsLocation(patternLocation) ? patternLocation.id : location.id}
             onClick={this._onClick}
             opacity={0.6}
             positions={groupedLocationShapePts[key]}
-            ref={`${patternLocation.id || patternLocation.locationId}`}
+            // $FlowFixMe Flow doesn't understand patternHaltIsLocation
+            ref={`${patternHaltIsLocation(patternLocation) ? patternLocation.id || patternLocation.locationId : location.id}`}
             tooltip={`${index + 1}`}
             zIndexOffset={active ? 1000 : 0}
           >
             <Popup>
               <div // popup requires single child (i.e., single div)
                 style={{minWidth: '240px'}}>
+                {!patternHaltIsLocation(patternLocation) && <h4>Location Group</h4>}
                 <h5>{stopName}</h5>
-                <Row>
-                  <Col xs={12}>
-                    <PatternStopButtons
-                      {...this.props} patternStop={patternLocation} stop={location} />
-                  </Col>
-                </Row>
-                <Row>
-                  <Col xs={6}>
-                    <FormGroup
-                      controlId='defaultTravelTime'>
-                      <ControlLabel>Travel time</ControlLabel>
-                      <MinuteSecondInput
-                        seconds={patternLocation.flexDefaultTravelTime}
-                        onChange={this._onChangeTravelTime} />
-                    </FormGroup>
-                  </Col>
-                  <Col xs={6}>
-                    <FormGroup
-                      controlId='defaultTimeInLocation'>
-                      <ControlLabel>Time in location</ControlLabel>
-                      <MinuteSecondInput
-                        seconds={patternLocation.flexDefaultZoneTime}
-                        onChange={this._onChangeTimeInLocation} />
-                    </FormGroup>
-                  </Col>
-                </Row>
+                {patternHaltIsLocation(patternLocation) && <React.Fragment>
```

Comment:
Should something else be shown if this is not a location?

Reply:
Sort props, and I would reformat the tags here so that opening and closing brackets are on the same indent (same with `PatternStopMarker`).